requirements.txt

```diff
@@ -4,7 +4,7 @@
 bracex==2.3.post1 ; python_version >= '3.7'
 cairocffi==1.4.0 ; python_version >= '3.7'
 cairosvg==2.5.2
-certifi==2022.12.7 ; python_version >= '3.6'
+certifi==2022.9.24 ; python_version >= '3.6'
```

Comment:
Hmm weird, this looks like it regressed. Would just revert this file.

Reply:
Done